test/helpers/workflow.js

```diff
@@ -28,13 +29,15 @@ module.exports = {
   },
 
   destroy: () => {
-    return knex.destroy();
+    return Promise.resolve()
+      .then(() => knex.destroy())
+      .then(() => workflow.flow.db.destroy());
```

Comment:
I'm guessing this line was the crux of the database connection issue?

Reply:
I'm not certain this will actually fix the issue reliably given the way it's currently scoped though.

The `workflow` and `knex` variables are scoped to the top level of the module, and so every call to create will reassign them, creating new connections each time, but the `destroy` call will only tear down the most recently created.

Reply:
ORLY yeah this "fixes" the issue, in that it actually runs them all